.rubocop_todo.yml

```diff
@@ -1,17 +1,60 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2019-07-10 00:42:40 +0000 using RuboCop version 0.71.0.
+# on 2020-04-25 17:05:54 +0000 using RuboCop version 0.82.0.
```

Comment:
There's a dilemma.
- The regeneration of this file hides tons (100's, maybe 1,000's) of offenses: 
  - new cops added between 0.71.0, 0.82, both existing and future offenses;
  - old cops, some only for existing offenses, some for future offenses.
- On the other hand, not regenerating would create tons of issues to be resolved in this PR, which would delay rollout of 2.6.6 and make the PR harder to review.